build.gradle.kts

```diff
@@ -17,41 +15,47 @@ repositories {
     mavenCentral()
 }
 
-dependencies {
-    api("org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1")
-    testImplementation(kotlin("stdlib"))
-    testImplementation(kotlin("test"))
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.9.0")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.9.0")
-}
-
-java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
-    withJavadocJar()
-    withSourcesJar()
-}
-
-tasks {
-    withType<KotlinCompile>().configureEach {
-        this.kotlinOptions {
-            jvmTarget = "1.8"
-            freeCompilerArgs = listOf("-opt-in=kotlinx.serialization.ExperimentalSerializationApi")
+kotlin {
+    jvm {
```

Comment:
I ran `publishToMavenLocal` and there's no JAR with .class files. Is this expected? Can the library still be consumed by non-KMP projects?

Reply:
tested locally:

Reply:
It's absent in `io.github.detekt.sarif4k:sarif4k:0.3.0-SNAPSHOT`, but presented in `io.github.detekt.sarif4k:sarif4k-jvm:0.3.0-SNAPSHOT`. non-KMP projects have to change dependency to `sarif4k-jmv` from `sarif4k`.
I guess it's a major change for you...

Reply:
I think this is expected in the 0.2.0 version range :)

The only known user is Detekt (using Gradle) for this lib I think, so there are no guarantees here for others.

I used it in my personal project too, I think it's an acceptable change considering the benefits. @chao2zhang will be weigh in more.

Reply:
will try to rename `sarif4k-jvm` to `sarif4k` and `sarif4k` to `sarif4k-multiplatform` for backward compatibility

Reply:
I don't think it's worth it. I tested it on my project which uses this lib. I added `mavenLocal()`, bumped the version, and proved that and referencing `io.github.detekt.sarif4k:sarif4k` pulls in the right dependencies even without KMP (it's an included build `gradle/plugins` with plugins: `kotlin-dsl`, `java-gradle-plugin`.)

So Gradle users won't notice a thing. Others (`.main.kts` or Maven) might need to adjust the artifacts.

Reply:
cool, thanks!